### PR TITLE
Update logging.md

### PR DESCRIPTION
**Pull request by ravi16a87**
Proposed changes that explain how to use dependency injection using ILogger in the different classes with Azure Application Insights.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=26155) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=26155) before we can accept your contribution.<br/><br/>:x: ravi16a87 [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=26155)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=26155) it.</sub>

File: docs/core/extensions/logging.md [hunk 1]
```diff
@@ -525,6 +525,58 @@ class Program
...
+                    builder.AddApplicationInsights("Azure Application Insights Key");
+                });
+                services.AddSingleton<App>(x =>                {
+                    ILogger<App> logger = x.GetRequiredService<ILogger<App>>();
+                    return new App(logger);
+                });              
+                serviceProvider = services.BuildServiceProvider();                
+                serviceProvider.GetService<App>().Run();               
+            }       
+}
```
Comment (IEvangelist) [inline docs/core/extensions/logging.md:562]:
```suggestion
class Program
{
    static void Main(string[] args)
    {
        using var channel = new InMemoryChannel();
        IServiceCollection services = new ServiceCollection(); 
        services.Configure<TelemetryConfiguration>(config => config.TelemetryChannel = channel);
        services.AddLogging(builder =>  builder.AddApplicationInsights("<AzureAppInsightsKey>"));
        services.AddSingleton<App>(sp =>
        {
              ILogger<App> logger = sp.GetRequiredService<ILogger<App>>();
              return new App(logger);
        });

        var serviceProvider = services.BuildServiceProvider();                
        serviceProvider.GetService<App>().Run();
    }
}
```

Fix formatting

File: docs/core/extensions/logging.md [hunk 2]
```diff
@@ -525,6 +525,58 @@ class Program
...
+        public App(ILogger<App> logger)
+        {
+            this._logger = logger;
+        }
+
+        public void Run()
+        {
+            this._logger.LogInformation("Example log message");            
+        }
+    }
```
Comment (IEvangelist) [inline docs/core/extensions/logging.md:577]:
```suggestion
 using Microsoft.Extensions.Logging;
 
 class App
 {
     private readonly ILogger _logger;
     
     public App(ILogger<App> logger)
     {
         _logger = logger;
     }

    public void Run()
    {
        _logger.LogInformation("Example log message");            
    }
}
```

**Review by IEvangelist — CHANGES_REQUESTED** [review]
While there is a lot of clean up around the way this code is formatted and presented, I still do not think it belongs in this article. It should be part of the Azure Application Insights content.

Please consider creating a pull request in this repo:

https://docs.microsoft.com/en-us/azure/azure-monitor/

**Comment by IEvangelist** [comment]
Hi @ravi16a87
I understand the reasoning for this pull request, but I believe it is better suited in the Azure repo instead.  

I'm going to close this PR, but please consider proposing changes similar to this in the Azure Application Insights docs. https://github.com/MicrosoftDocs/azure-docs

**Comment by ravi16a87** [comment]
Hi @IEvangelist , Thank you for your time. I appreciate your comments.

**Pull request closed without merging**